### ISSUE TITLE
Download and install Nexus Mods App

### ISSUE THREAD

**Issue by Greg-Nexus** [Nexus-Mods/NexusMods.App#376]
# User story

**As a** user

**I want to** download the Nexus Mods App

**So that** I may install it

## Requirements
Users will need to be able to download the app and install it without relying on CLI tools.

The scope of this issue is basic:

- Make a downloadable file available
- Download the file
- Install the app to a chosen directory/location

The above needs to be available for Windows.

Out of scope (to be done later, in separate issues):

- App icons (needs design)
- Pretty installation process/wizard/splash screen
- Hosting on Nexus Mods
- Linux Installer

## Design
N/A

## Related Issues

- **Blocks:** https://github.com/Nexus-Mods/NexusMods.App/issues/444
